src/main/webapp/karma.conf.js

```diff
@@ -29,7 +29,14 @@ module.exports = function(config) {
 
     frameworks: ['jasmine', 'requirejs'],
 
-    browsers: ['ChromeHeadless'], // or 'Chrome'
+    browsers: ['ChromeHeadlessNoSandbox'], // or 'Chrome'
```

Comment:
What is the `// or 'Chrome'` comment trying to tell me here, and is whatever it's trying to tell me still true with this change?

Reply:
It came along from the karma examples.
It's a reminder that tests can be run visually or headless.